run/helloworld/app.rb

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START run_hello_service]
+require "sinatra"
+
+set :bind, "0.0.0.0"
+port = ENV["PORT"]
```

Comment:
If port is unspecified, does this fall back to 8080 implicitly? If not we prefer to explicitly set the default rather than have the sample fail for local use.

Reply:
It does, but I will add the OR statement to specify 8080.